### DISCUSSION TITLE
Request for Assistance with Two-Factor Authentication

### DISCUSSION THREAD

**Discussion by Kaleb11402** [community/community#147912] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Dear GitHub Support,

I am writing to request assistance with accessing my GitHub account. I know my username and password, and I can log in successfully; however, I am unable to complete the two-factor authentication (2FA) process.

Here are the details of the issue:

I have 2FA enabled on my account and had it configured with the Microsoft Authenticator app ([Google Play link](https://play.google.com/store/apps/details?id=com.azure.authenticator&pcampaignid=web_share)).
After logging in with my username and password, I am prompted to enter the TOTP code from the authenticator app.
The app generates a TOTP code, but when I enter the code, it says, "Two-factor authentication failed."
Unfortunately, I lost my laptop due to theft two months ago, and with it, I lost my recovery codes and passkey.
To demonstrate that I have correctly set up GitHub 2FA on the Microsoft Authenticator app.

I have many important projects stored in my GitHub account, and I am unable to access them. Could you please help me reset the two-factor authentication on my account so I can regain access?

I tried to send an email to [[support@github.com](mailto:support@github.com)], but it failed because they have blocked incoming messages. What should I do now?

Thank you for your time and assistance. I am happy to provide any further information you might need to verify my identity.

**Comment by davevad93** [comment, marked as answer]
Hi @Kaleb11402 ,

For account related issues it's recommended to contact support by opening a ticket [here](https://support.github.com/contact/account), select "`Two-factor authentication recovery`" and explain your problem.

**Reply by Kaleb11402** [reply]
Ok thank you @davevad93.

**Comment by shinybrightstar** [comment]
Hi @Kaleb11402  👋🏾,It looks like @davevad93 's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)

**Reply by Kaleb11402** [reply]
I did

**Comment by MasterInQuestion** [comment]
<&~~>Claimed current bug of Microsoft Authenticator?
    https://github.com/orgs/community/discussions/147499#discussioncomment-11620449 </&>
    Seems to be Authy bug...
